### DISCUSSION TITLE
Ordering subtasks question

### DISCUSSION THREAD

**Discussion by bobykvv** [TechbeeAT/jtxBoard#1335] [Q&A]
Hello! 

There was a discussion on close topic already, however I think it didn't come to the end...

Problem is that I (like many others, I bet) use subtasks for creating something like shopping lists, and if one's been doing that for some time, usually newer positions are in a higher priority for the creator. And I find it convenient if they stay at the top of subtasks. But I'd prefer completed subtasks to be moved to the end of the list. OR be able to turn on a filter to make them disappear/appear like we have in the "main" tasks. 

For now I don't see a way to do that in the application. Or I missed something? 
Either I can sort them out in a way that I see newer subtasks on the top, but those completed also stay on the top, or all of them are sorted in a reverse order when oldest subtasks are on top and newest appear in the end together with completed.

P.S. Anyway, the app is absolutely great! Kudos to the author!

**Comment by patrickunterwegs** [comment]
Closing this discussion as we anyway have the follow-up in issues (#1339)
